core/domain/skill_domain.py

```diff
@@ -803,14 +803,14 @@ def update_worked_examples(self, worked_examples):
         """Updates the worked examples list of the skill.
 
         Args:
-            worked_examples: list(dict). The new worked examples of the skill.
+            worked_examples: list(SubtitledHtml). The new worked examples of
+                the skill.
         """
+
```

Comment:
We usually exclude blank lines both before and after docstrings. This is defined [here](https://www.python.org/dev/peps/pep-0257/#id17) In PEP 257, which defines the canonical docstring style conventions

Reply:
Done!

Reply:
The summary line of a docstring should always fit on one line since some tools assume this. From [PEP 257](https://www.python.org/dev/peps/pep-0257/#id17):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

You can add as many lines as you like between the summary line and the arguments, though, for example:

```python
def my_func(param):
    """This line summarizes what the function does.

    Here are some more details about the function.

    Arguments:
        param: Parameter for the function that should be ...

    Returns:
        Whatever the function returns.
    """"
```

Reply:
My bad, multi-line summaries appear to be standard for this file, so I'm resolving this comment.